src/App.jsx

```diff
@@ -1,5 +1,29 @@
-import { useState } from "react";
+import { useState, useEffect } from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { getArticles } from "./utils/api";
+import Header from "./components/Header";
+import Home from "./components/Home";
+import Articles from "./components/Articles";
+import "./styles.css";
 
-function App() {}
+function App() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((res) => {
+      setArticles(res);
+    });
+  }, []);
+
```

Comment:
I would move the state and useEffect into the Articles.jsx component. There is no need for them to be in App. Remember, whenever state changes, the component will re render. There s no need for the whole app to pre-render if the state of articles changes 🙂